types/gettext.js/index.d.ts

```diff
@@ -24,8 +24,13 @@ declare namespace i18n {
     }
 
     interface Gettext {
-        setMessages(domain: string, locale: string, messages: JsonDataMessages, plural_forms?: PluralForm): Gettext;
-        loadJSON(jsonData: JsonData, domain?: string): Gettext;
+        setMessages(
+            domain: string,
+            locale: string,
+            messages: JsonDataMessages,
+            plural_forms?: string | undefined,
```

Comment:
Nitpick: having `| undefined` is redundant for optional function parameters. (Function parameters are not affected by the `exactOptionalPropertyTypes` config option)

```suggestion
            plural_forms?: string,
```

Reply:
Fixed.